dev/io.openliberty.wsoc.21.internal_fat/fat/src/io/openliberty/wsoc/FATSuite.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.wsoc;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.ibm.ws.fat.util.FatLogHandler;
+
+import componenttest.rules.repeater.EmptyAction;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.RepeatTests;
+import io.openliberty.wsoc.tests.BasicTest;
+
+/**
+ * Collection of all example tests
+ */
+@RunWith(Suite.class)
+/*
+ * The classes specified in the @SuiteClasses annotation
+ * below should represent all of the test cases for this FAT.
+ */
+@SuiteClasses({
+                BasicTest.class
```

Comment:
This test is also run in the existing wsoc FAT isn't it? Is the plan just to keep this here until we have some 2.1 specific tests and then remove it?